src/test/resources/fixtures/kotlin/build.gradle.kts

```diff
@@ -21,6 +21,7 @@ node {
     yarnVersion.set("")
     npmInstallCommand.set("install")
     distBaseUrl.set("https://nodejs.org/dist")
+    allowInsecureProtocol.set(null as Boolean?)
```

Comment:
if `null` is the default, shouldn't we skip setting it?

Reply:
I'm easy either way - I added it in the interests of illustrating the flag, but I've no objection to removing it and allowing it to default to null.